app/qml/main.qml

```diff
@@ -895,6 +901,10 @@ ApplicationWindow {
       storageLimitDialog.open()
     }
 
+    function onMonthlyContributorLimitReached( uploadSize ) {
```

Comment:
We do not use this `uploadSize` anywhere. What would be nice though is to include the actual limit - similarly to what we did on plugin (see https://github.com/MerginMaps/qgis-plugin/pull/640/commits/f1ea460fb94c5c45d1d4e88218bbbd451dad624b#diff-d5f841eda10b7504474461cbece2f5a718c04995de1956f64f1674138655c675R28)

```suggestion
    function onMonthlyContributorLimitReached( contributorsQuota ) {
```

Reply:
Added in the last commit, similarly as the plugin